src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs

```diff
@@ -478,7 +478,30 @@ public static IEnumerator GetEnumerator(IsolatedStorageScope scope)
             // (say, for a particular StrongName). You could iterate any store opened at least once by NetFX on
             // NetFX as it would create the needed identity file. You wouldn't be able to iterate if it was only
             // ever opened by CoreFX, as the needed file isn't there yet.
-            throw new PlatformNotSupportedException();
+            return new IsolatedStorageFileEnumerator();
```

Comment:
I understand the desire to keep things similar.  At the same time, part of the point of having a separate codebase is to enable these kinds of changes.  I don't think it's worth keeping around an unnecessary class purely to help compare the codebases.